.editorconfig

```diff
@@ -8,6 +8,8 @@ indent_size = 2
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
+quote_type = single
```

Comment:
please don't commit edits to this file.

Reply:
Looks like lint-staged is not configured properly. Committed code is not reformatted. This is the main problem why these changes are added.